src/index.js

```diff
@@ -0,0 +1,206 @@
+import './style.css';
+import 'bootstrap/dist/css/bootstrap.css';
+import update from './update';
+import { createTodo, destroyTodo, updateTodo } from './todos_controller';
+
+const button = document.querySelector('button');
+
+class Todo {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let toDos = [];
+
+function createTodoItem(todo) {
+  const li = document.createElement('li');
+  li.innerHTML = `
+    <div class="flex todo-element">
+      <div>
+          <input type="checkbox" class="checkbox"
+          ${todo.completed ? 'checked' : ''}>
+          <span>${todo.description}</span>
+      </div>
+      <span class="material-icons edit-icon" style="cursor: pointer">
+          more_vert
+      </span>
+    </div>
+    <hr>`;
+  return li;
```

Comment:
- [ ] You can use the arrow function instead of the regular function because it allows implicit return.

- [ ] You can keep your code DRY by not repeating your code more than once.